.github/workflows/pr.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     env:
       BUILDKIT_PROGRESS: plain
       DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-latest-4-cores
```

Comment:
Depending on the changed files, yeah it will build docker images. For example, when fetch_versions creates a PR, that will trigger a build again with the new plugins.

Reply:
Yep, got it 👍